### ISSUE TITLE
Système d'ajout de partenaires

### ISSUE THREAD

**Issue by PEM--** [PEM--/asv-la-soiree#30]
Créer un système d'ajout de partenaire dans le backoffice avec une classification à 3 niveaux :
* Partenaires Or
* Partenaires Argent
* Partenaires

**Comment by PEM--** [comment]
Par défaut, le serveur ne présente aucun partenaire.

La première étape consiste à préparer ses logos. Ce sont des PNG en 120x120 dont le fond est blanc. Je conseille fortement de les optimiser grâce à un logiciel de type [ImageOptim](https://imageoptim.com/fr.html).

Il faut ensuite créer les partenaires sur le back-office dans la catégorie **Partenaires** :
![capture d ecran 2015-07-28 a 20 03 08](https://cloud.githubusercontent.com/assets/227831/8939303/e030403e-3563-11e5-81d3-428463d38f6d.png)

Pour chaque logo téléchargé, il est nécessaire de :
* Ajouter un nom : celui de la société ou de l'association.
* Ajouter l'adresse de son site web ou de sa page FaceBook.
* Sélectionner une catégorie dans laquelle le logo apparaîtra : gold, silver, bronze
* Télécharger une des images préparées (voir étape précédente).
* Enregistrer : La modification est immédiatement affichée sur les pieds de pages mobiles et navigateurs.

![capture d ecran 2015-07-28 a 20 21 51](https://cloud.githubusercontent.com/assets/227831/8939710/5a23244a-3566-11e5-9f5d-1f1281b702ad.png)

À tout moment, vous pouvez modifier les entrées des partenaires, changer leurs images, les passer d'une catégorie à l'autre, ...

**Comment by JulietteASV** [comment]
Coucou PEM,

Je suis entrain de tester ce process, c'est vraiment cool ! Bon ne regarde pas la qualité des images, c'est juste que j'attends les fichiers des partenaires... Par contre, quand je supprime des partenaires (malheureusement, tu as créé plus d'entrées que de partenaires réels, à mon grand regret lol) et bien je suis renvoyée directement sur la page des inscrits.. 
C'est pas bloquant car je retourne en un clic sur la page des partenaires.. mais si on peut fixer ça... ^^

Merci

**Comment by PEM--** [comment]
Oui, il y un pépin sur la suppression des éléments en base sur l'interface d'administration du CMS. Je ne l'ai pas encore remonté à Nicolas Lopez. Le fonctionnel reste bon donc pour l'instant c'est très mineur. Ce sera plus dérangeant sur EVAFORM. Mais d'ici là, on sera sur une autre version plus aboutie.

**Comment by JulietteASV** [comment]
Oui oui pas de soucis, merci

**Comment by JulietteASV** [comment]
PEM, peux-tu aller checker l'affichage des logos sur le site de test ? Je n'en suis pas très satisfaite, pas de la qualité des images hein, mais de l'emplacement des logos ... Il y en a qui sont plus haut que d'autres, et aussi tu crois que je pourrais avoir la main sur l'ordre d'apparition de ces logos ? Je voudrais les ranger par ordre alphabétique car il y a des partenaires qui viennent des fois se plaindre car ils ne comprennent pas pk ils apparaissent après un tel ou un autre... Donc en les rangeant dans leur catégorie par ordre alphabétique, ils n'ont plus rien à dire ^^

**Comment by PEM--** [comment]
En effet, tes images ne font pas toutes 120x120. Cela explique tes problèmes d'alignements. J'ai rappelé les dimensions sur l'interface pour que tu y penses à l'insertion.

Actuellement, la règle est de les afficher dans l'ordre d'insertion. Tu peux facilement effacer, puis réajuster chaque catégorie.

**Comment by PEM--** [comment]
N'oublie pas non plus d'aplatir tes images sur fond blanc et de les optimiser (1ère étape de la doc :wink: ).

**Comment by JulietteASV** [comment]
Oui ok pas de soucis.. ^^



2015-07-31 11:24 GMT+02:00 Pierre-Eric Marchandet <notifications@github.com>
:
